channels/jaicp/src/main/kotlin/com/justai/jaicf/channel/jaicp/polling/PollingChannel.kt

```diff
@@ -5,5 +5,18 @@ import com.justai.jaicf.channel.jaicp.JaicpBotChannel
 internal data class PollingChannel(
     val url: String,
     val botChannel: JaicpBotChannel,
-    val poller: RequestPoller
-)
\ No newline at end of file
+    val poller: RequestPoller,
+) {
```

Comment:
I think we can get rid of isActive/start/stop logic in PollingChannel and move it to Dispatcher where it's used to be in order to simplify code flow a little bit